lib/jellyfishService.js

```diff
@@ -153,60 +164,72 @@ function createServer(serverConfig, mongoClient, seagullClient, userApiClient, g
   app.use(bodyparser.json({ limit: '4mb' }));
   app.use(errorHandler({ dumpExceptions: true, showStack: true }));
 
-  app.get('/info', function(request, response) {
-    log.info('Handling versions request');
+  app.get('/info/?:userId?', function (request, response) {
+    
+    var uploaderDestination = null;
+
+    if (request.params.userId) {
+      if (serverConfig.uploader.platformUsers.includes(request.params.userId.trim().toLowerCase())) {        
+        uploaderDestination = 'platform';
+      }else{
+        uploaderDestination = 'jellyfish';
+      }
+      log.info(`uploader destination for user with id '${request.params.userId}' is '${uploaderDestination}'`);
+    }
 
     var body = {
       auth: {
         realm: schemaEnv.authRealm,
         url: schemaEnv.authUrl,
       },
       versions: {
-        uploaderMinimum : schemaEnv.minimumUploaderVersion,
+        uploaderMinimum: schemaEnv.minimumUploaderVersion,
+        uploaderDestination,
```

Comment:
Should we pull this out of the `versions` block and just put it at the top-level? Doesn't seem like it is related to a version. WDYT?